Demos/Blazorise.Demo/Pages/Dashboard.razor

```diff
@@ -1,30 +1,29 @@
 ﻿@page "/"
 @inject IVersionProvider VersionProvider
-<Heading Size="HeadingSize.Is1" Margin="Margin.Is3.FromBottom">Blazorise</Heading>
 
-<Paragraph>
-    Blazorise is a component library built on top of Blazor and CSS frameworks like <Anchor To="https://getbootstrap.com/" Target="Target.Blank">Bootstrap</Anchor>, <Anchor To="https://bulma.io/" Target="Target.Blank">Bulma</Anchor>, <Anchor To="https://ant.design/" Target="Target.Blank">Ant Design</Anchor>, and <Anchor To="http://daemonite.github.io/material/" Target="Target.Blank">Material</Anchor>. It can be used to build responsive, single-page web applications.
-</Paragraph>
+<PivotGrid TItem="Employee" Data="inMemoryData">
+    <PivotGridColumns>
+        <PivotGridColumn Caption="Year"></PivotGridColumn>
+        <PivotGridColumn Caption="Quarter"></PivotGridColumn>
+    </PivotGridColumns>
+    <PivotGridValues>
+        <PivotGridValue Caption="Quantity"></PivotGridValue>
+        <PivotGridValue Caption="Amount"></PivotGridValue>
+    </PivotGridValues>
+    <PivotGridRows>
```

Comment:
What is the use-case for rows? Why do we need to define them?

Reply:
I understand now what you wanted to do. It has an API similar to the SF.

Reply:
All the same uses cases as this API
enum Area {
  Row, // it will be shown in a row header and grouped by this => PivotGridRow
  Column,  // it will be shown as a regular column, just like DataGrid => PivotGridColumn
  Data, // it will be used for the summary => PivotGridValue
}